server/tests-py/validate.py

```diff
@@ -345,6 +346,9 @@ def equal_CommentedMap(m1, m2):
                     for (k1,v1),(k2,v2) in zip(m1_l,m2_l)))
     # else this is a scalar:
     else:
+        if isinstance(m1, str) and isinstance(m2, str):
```

Comment:
I don't think we should introduce the correctness of a test based on unpredictable output. Wasn't the discussion in Slack to either skip this for PG13 or introduce a list of outputs (and if any one matches, then succeed)?

Reply:
Have modified currently available functions to look for `allowed_responses` as well. Also, it turns out that there was only one test where this was happening.